test/04_calendar_date_overrides.js

```diff
@@ -335,7 +457,7 @@ describe('Testing national calendar overrides', function() {
   describe('Our Lady of Sorrows', function() {
 
     it('Should be celebrated on the 15th of September 2018 as a memorial in the General Calendar', function() {
```

Comment:
`on the 15th of September 2018` → `on 15 September 2018` (or `September 15, 2018`)